ldm/dream/args.py

```diff
@@ -485,6 +485,19 @@ def _create_dream_cmd_parser(self):
             type=str,
             help='Directory to save generated images and a log of prompts and seeds',
         )
+        render_group.add_argument(
+            '-save_progress',
```

Comment:
single-dash arguments are supposed to be used with single-letter flags

this file already has plenty of arguments so maybe you should skip the single-dash arguments entirely

Reply:
see https://serverfault.com/a/387936 for more info